### ISSUE TITLE
Calculation of MSR in Addition Phase

### ISSUE THREAD

**Issue by ShokofehVS** [ShokofehVS/SecBic-CCA#10]
Victor (@padilha), I want to know if you possibly can help me find out some parts in `[cca.py](https://github.com/padilha/biclustlib/blob/master/biclustlib/algorithms/cca.py)` for calculating MSR score in addition of node:

It's a general question but I want to be sure about that. Are we using `sub_data_rows = data[rows]` (line 198) and  `sub_data_cols = data[:, cols]` (line 214) because these nodes are particularly being added to intermediate bicluters?

**Comment by padilha** [comment]
In the first case, sub_data_rows is used to calculate the mean of each column node (see line 202 and step 2 of Algorithm 3 in the original paper) that can potentially be added to the intermediate bicluster. Next, the MSR of each candidate column node is calculated in lines 204-206.

Similarly, for _calculate_msr_row_addition, sub_data_cols is used to calculate the mean of the row nodes (see line 217 and step 4 of Algorithm 3 in the original paper) that can potentially be added to the intermediate bicluster. Next, the MSR of each candidate row node is calculated in lines 220-222 and 224-226 (for inverse rows).

To sum up, these nodes are not being added yet. We are just calculating their MSR. We check if they are going to be added in lines 166-167 and 171-173.
